src/lib.rs

```diff
@@ -285,20 +379,33 @@ where
     }
 
```

Comment:
We seem to be missing a `notify_get()`. Is that intentional and if so what is the reason ? `get()` seems to remove expired items so it would seem to make sense for a `notify_get()`.